### DISCUSSION TITLE
Improving interaction with the bot

### DISCUSSION THREAD

**Discussion by dupuyarc** [noop-inc/documentation#123] [Feature Requests and Ideas]
[_I have split this out from the original typo bug report in #122, since this is more of a feature request._]

When trying to report this minor bug, I selected "I have a feature request" and I was redirected to the GitHub organization discussions and ended up being able to create this bug report. But before I did that, I used "I have a question" (since this was a bug, not a feature request) and got some really rather wordy responses from the bot:

> I found a small typo in the Engagement popup

> I understand that you found a small typo in the Engagement popup, but the information I've found about your query doesn't seem to directly address this issue. I apologize for the uncertainty, and I'll do my best to provide a partial answer based on the available sources.
> One possible interpretation of your query is that you're looking for a way to report the typo you found. Unfortunately, the sources don't provide a direct method for reporting typos in the Engagement popup. However, you could try reaching out to Noop Inc through their community or by contributing to their documentation, as mentioned in one of the sources .
> If you need assistance with the Engagement popup itself or have questions about its functionality, please provide more context or information about the issue you're facing, and I'll do my best to help you based on the available sources.
> Do you have any specific questions about the Engagement popup or its features? Providing more context or details about the typo you found might help me better understand your query and assist you further.

There's an awful lot of words there, and no clickable links that might take me where I want to go. Perhaps there is some prompt engineering you can do in the pre-conversation that might suggest to the LLM to be more concise, and to offer link URLs where possible.

I would also suggest providing a third possibility for the initial engagement handling: "I may have found a bug", in addition to the question and feature request choices.

**Comment by 1800joe** [comment]
Great points, we'll look into better options. Thanks!
